.github/workflows/release_test.yml

```diff
@@ -1,9 +1,9 @@
 name: Release Test
 
 on:
+    workflow_dispatch:
     push:
         branches:
-            - feature/**
```

Comment:
Is it possible to make the Release workflow dependent on this one, so that we don't have to remember to check it when we do a CCI release?

Reply:
See 3f09e8f